### PR TITLE
Docs: cactbot customization Korean translation

### PR DESCRIPTION
**Pull request by Jaehyuk-Lee**
Resolved: ~~I tried to make markdown links work which includes `:` in title, HTML special character, URL encoded character did not work. Anyone knows how to make it work?~~

To see how translated page is displayed in markdown, go to [this page](https://github.com/Jaehyuk-Lee/cactbot/blob/docs-ko/docs/ko-KR/CactbotCustomization.md).

## Typo Suspicion
"[This folder](https://github.com/quisquous/cactbot/tree/main/docs) has example customization files." in [cactbotCustomization.md](https://github.com/quisquous/cactbot/blob/main/docs/CactbotCustomization.md) right?
I guess it's typo.

## Concerning Korean Translation

1. 정확히하자면, 외래어 표기법에 따르면`디렉터리`가 맞지만 일반적으로 `디렉토리`를 사용합니다. (네이버 사전(옥스포드 영한사전 사용)도 `디렉토리`를 사용합니다.)  
Actually, `디렉터리` is right in foreign language notation. it usually uses `디렉토리`. (Oxford English-Korean dictionary uses `디렉토리`)
2. `override`에 적당한 한국어 번역을 찾지 못했습니다. 대신 `덮어쓰기`를 사용했는데 더 좋은 아이디어가 있다면 꼭 말해주세요.  
There's no appropriate translation for `override` in Korean, I used `덮어쓰기` which is `overwrite`. If you have any better idea, please tell me.

### CODE REVIEW

**Comment by MaikoTan** [comment]
I think it is better to add a link to this document in the English version.

Just like

`Other Languages: [한국어](./ko/CactbotCustomization.md)`

**Comment by Jaehyuk-Lee** [comment]
It would be better to add `readme.md` for each language shortcuts in `docs` folder when there's much more translations.

**Comment by Jaehyuk-Lee** [comment]
Translation is done, but I'll receive Korean user's suggestions for this first, then change status to "Ready to review".

File: docs/CactbotCustomization.md [hunk 1]
```diff
@@ -1,13 +1,15 @@
...
 - [User Directory Overview](#user-directory-overview)
 - [Setting Your User Directory](#setting-your-user-directory)
 - [Customizing Appearance](#customizing-appearance)
 - [Overriding Raidboss Triggers](#overriding-raidboss-triggers)
-  - [Example 1: changing the output text](#example-1--changing-the-output-text)
-  - [Example 2: making provoke work for all jobs](#example-2--making-provoke-work-for-all-jobs)
-  - [Example 3: adding custom triggers](#example-3--adding-custom-triggers)
+  - [Example 1: changing the output text](#example-1:-changing-the-output-text)
+  - [Example 2: making provoke work for all jobs](#example-2:-making-provoke-work-for-all-jobs)
+  - [Example 3: adding custom triggers](#example-3:-adding-custom-triggers)
```
Comment (panicstevenson) [inline docs/CactbotCustomization.md:12]:
```suggestion
  - [Example 1: changing the output text](#example-1-changing-the-output-text)
  - [Example 2: making provoke work for all jobs](#example-2-making-provoke-work-for-all-jobs)
  - [Example 3: adding custom triggers](#example-3-adding-custom-triggers)
```
TOC links don't have punctuation

Comment (Jaehyuk-Lee) [reply docs/CactbotCustomization.md]: Oh wow that works! thanks!

**Comment by Jaehyuk-Lee** [comment]
> `Other Languages: [한국어](./ko/CactbotCustomization.md)`
I've removed `Other Languages` part and just show an globe icon.

I think
&#x1F310; 한국어 / 日本語 ... and so on is better looking.

**Comment by panicstevenson** [comment]
This is purely a discussion of taste, so please don't feel obligated to do it:
How do you feel about always having the languages at the header, including the file you're in?
English:
```md
🌎 [en] [[ko](./link/to/ko.md)]
```
> 🌎 [en] [[ko](./link/to/ko.md)]

Korean:
```md
🌎 [[en](./link/to/en.md)] [ko]
```
> 🌎 [[en](./link/to/en.md)] [ko]

**Comment by Jaehyuk-Lee** [comment]
I understand the idea, what about

English:
```
🌎 [**English**] [[한국어](./link/to/ko.md)]
```
🌎 [**English**] [[한국어](./link/to/ko.md)]

Korean:
```
🌎 [[English](./link/to/en.md)] [**한국어**]
```
🌎 [[English](./link/to/en.md)] [**한국어**]

Since it's better to see in their own languages.

**Comment by panicstevenson** [comment]
Definitely, whatever you think works best; I'm not tied to any idea, it just felt odd I couldn't navigate "back" to another language (if I picked the wrong one or something)

**Comment by MaikoTan** [comment]
Can you change `ko` to `ko-KR`? I think I can add `zh-CN` (for Chinese mainland players) and `zh-TW` (for Taiwan players) when adding Chinese translation in the future. Although they both use Chinese, many games and computer fields use different terms.

**Comment by Una-p** [comment]
1. 사실 어떻게 써도 큰 상관은없어보입니다! 표준어상 디렉터리가 맞는말이긴 한데 둘다 쓰는 단어니까요..
2. 오버라이드 자체가 코딩용어라가지고, 기능상 의역하는게 좋을거같아요. 덮어쓰기도 괜찮겠네요!

**Comment by Jaehyuk-Lee** [comment]
@Una-p 그 부분은 #1869 에서 다루고 있습니다. 애초에 이 문서를 굳이 번역하게 된 이유가 원래 수정하기 쉬웠는데 그게 불가능해져서 그나마 한국어 가이드가 있어야지 수정을 할 수 있을 것 같아서요.
번역하다보니 원문 내용을 담으려고 해서 장황하게 설명하는 듯한 문서가 되었는데, 그냥 간단히

```javascript
Options.Triggers.push({
  // 트리거 파일(.js) 최상단에 있는 ZoneId를 넣으세요.
  // 예시) ZoneId.MatchAll (모든 지역) 또는 ZoneId.TheBozjanSouthernFront.
  zoneId: ZoneId.PutTheZoneFromTheTopOfTheFileHere,
  triggers: [
      // 이 곳에 수정하고 싶은 트리거를 그대로 붙여넣고 원하는대로 수정하세요.
      // 아래는 절알렉에서 Nael Fireball 1 이라는 트리거를 붙여넣은 후 수정한 예시입니다.
      id: 'UCU Nael Fireball 1',
      netRegex: NetRegexes.ability({ source: 'Ragnarok', id: '26B8', capture: false }),
      netRegexDe: NetRegexes.ability({ source: 'Ragnarök', id: '26B8', capture: false }),
      netRegexFr: NetRegexes.ability({ source: 'Ragnarok', id: '26B8', capture: false }),
      netRegexJa: NetRegexes.ability({ source: 'ラグナロク', id: '26B8', capture: false }),
      netRegexCn: NetRegexes.ability({ source: '诸神黄昏', id: '26B8', capture: false }),
      netRegexKo: NetRegexes.ability({ source: '라그나로크', id: '26B8', capture: false }),
      delaySeconds: 35,
      suppressSeconds: 99999,
      infoText: {
        ko: '불 대상자 밖으로',
      },
      run: function(data) {
        data.naelFireballCount = 1;
      },
    },
  ],
});
```

이런식으로 알려주면 쉽지 않나 생각해요.

**Comment by Jaehyuk-Lee** [comment]
@Una-p 한번 관련 지식이 전혀 없는 사람들이 알아볼 수 있도록 "무작정 따라하기"를 넣어봤습니다. 번역본이 아니라 번역투가 없는 좋은 느낌도 있지만, 제가 처음부터 새로 작성했기 때문에 뭔가 설명이 보충되야 할지는 봐야할 것 같아요.

**Comment by Una-p** [comment]
오 제가 아직 이 방법대로 해보지않아서 라이브에 올라오면 한번해보려고했는데..! 수고하셨어요. 확실히 도움이 될거같네요!

**Comment by MaikoTan** [comment]
> @Una-p 한번 관련 지식이 전혀 없는 사람들이 알아볼 수 있도록 "무작정 따라하기"를 넣어봤습니다. 번역본이 아니라 번역투가 없는 좋은 느낌도 있지만, 제가 처음부터 새로 작성했기 때문에 뭔가 설명이 보충되야 할지는 봐야할 것 같아요.

If you think this part is worth putting into the document, I think it is better to add it to the English version as well, so that other language versions also have these instructions?

By the way, I am using crowdin for translating documents, although it'll break markdownlint ( and I don't know how to fix it ), but it is a wonderful website for sync different language version.

**Comment by quisquous** [comment]
> > @Una-p 한번 관련 지식이 전혀 없는 사람들이 알아볼 수 있도록 "무작정 따라하기"를 넣어봤습니다. 번역본이 아니라 번역투가 없는 좋은 느낌도 있지만, 제가 처음부터 새로 작성했기 때문에 뭔가 설명이 보충되야 할지는 봐야할 것 같아요.
> 
> If you think this part is worth putting into the document, I think it is better to add it to the English version as well, so that other language versions also have these instructions?

Yes, please.  If there are improvements to the original document, please send pull requests!

> By the way, I am using crowdin for translating documents, although it'll break markdownlint ( and I don't know how to fix it ), but it is a wonderful website for sync different language version.

Yeah, I haven't forgotten the request from long ago about using something like that for document translation.  I think I am just a bit overwhelmed with other things to investigate the options more thoroughly at this point, but I will continue to keep it in mind.  Breaking markdownlint is a bit frustrating, though.

**Comment by Una-p** [comment]
> Are there other changes needed here before this lands?

 We are trying to fix translationese. almost done :)

I saw many people used customized trigger. 
From modified in large numbers ( KO's TEA attack method is different about cactbot) to small changes.
In the latter case, even they don't know about any coding, just find what translated line and changed it to suit their taste.
> ex) find `리밋`[LB] and change `리미트 브레이크`[Limit Break].
New docs is for them that without background knowledge about programming.

**Pull request merged**